tests-legacy/Unit/Core/Cart/CartToOrder/CartToOrderTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace LegacyTests\Unit\Core\Cart\CartToOrder;
+
+use Configuration;
+use LegacyTests\Unit\Core\Cart\Calculation\Taxes\CartTaxesTest;
+use Order;
+use OrderCartRule;
+
+/**
+ * this class test the correct copy from cart to order
+ */
+class CartToOrderTest extends CartTaxesTest
```

Comment:
I know this is not the best way to get fixtures, it's the only one available for now
The fixture process is under heavy development and should be available soon